frontend/playwright.config.ts

```diff
@@ -0,0 +1,121 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+   testDir: './tests/playwright',
+   /* Maximum time one test can run for. */
+   timeout: 30 * 1000,
+   expect: {
+      /**
+       * Maximum time expect() should wait for the condition to be met.
+       * For example in `await expect(locator).toHaveText();`
+       */
+      timeout: 5000,
+   },
+   /* Run tests in files in parallel */
+   fullyParallel: true,
+   /* Fail the build on CI if you accidentally left test.only in the source code. */
+   forbidOnly: !!process.env.CI,
+   /* Retry on CI only */
+   retries: process.env.CI ? 2 : 0,
+   /* Opt out of parallel tests on CI. */
+   workers: process.env.CI ? 1 : undefined,
+   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+   reporter: [
+      [
+         'html',
+         {
+            open: 'never', // will not try to automatically open the report in the browser if test fails
+            outputFolder: 'tests/playwright/test-results/reports/',
+         },
+      ],
+   ],
```

Comment:
Just curious if we will ever need the `html` reporter? I think the list reporter might be faster (I'm just guessing this since it might take longer to generate the html). 

https://playwright.dev/docs/test-reporters#list-reporter

It looks like we can even change the reporter depending on the `process.env` like this `reporter: process.env.CI ? 'dot' : 'list'`.

Reply:
I think we can address this later on when we set up the workflow for Playwright.